components/actionmenu/stories/actionmenu.stories.js

```diff
@@ -50,40 +67,227 @@ export default {
 		},
 		packageJson,
-		docs: {
-			story: {
-				height: "200px",
-			}
-		},
+		metadata,
 		status: {
-			type: "unmigrated",
+			type: "migrated",
 		},
 	},
-	tags: ["unmigrated"],
+	tags: ["migrated"],
 };
 
 export const Default = ActionMenuGroup.bind({});
 Default.args = {
+	triggerArgs: {
+		iconName: "AddCircle",
+		label: "Add",
+	},
+	menuArgs: {
+		hasActions: true,
+		selectionMode: "multiple",
+		items: [{
+			heading: "Menu section header",
+			description: "Menu section description",
+			items: [{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			}],
+		}, {
+			heading: "Menu section header",
+			description: "Menu section description",
+			selectionMode: "none",
+			hasActions: false,
+			items: [{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},
+			{
+				label: "Menu item",
+				iconName: "Circle",
+			},],
+		}],
+	},
+};
+
+// ********* DOCS ONLY ********* //
+/**
+ * By default, the menu is opened by pressing the trigger element or activating it via the <kbd>Space</kbd> or <kbd>Enter</kbd> keys. However, there may be cases where the trigger should perform a separate action on press such as selection, and should only display the menu when long pressed. For this use-case, the menu will only be opened when pressing and holding the trigger or by using the <kbd>Option</kbd> (Alt on Windows) + <kbd>Down arrow</kbd>/<kbd>Up arrow</kbd> keys while focusing the trigger.
+ *
+ * This example illustrates the expected visuals and states of the action menu for a trigger with both long press and short press behaviors.
+ *
+ * Please note that the long press functionality is not implemented in this documentation and the example serves only as a visual reference.
+ */
+export const LongPress = Template.bind({});
+LongPress.args = {
+	position: "end-top",
+	hasLongPress: true,
 	isOpen: true,
-	position: "bottom",
-	label: "More actions",
-	iconName: "More",
-	items: [
-		{
-			label: "Action 1",
-		},
-		{
-			label: "Action 2",
-		},
-		{
-			label: "Action 3",
+	triggerArgs: {
+		iconName: "CropRotate",
+	},
+	menuArgs: {
+		customStyles: {
+			"--mod-menu-inline-size": "max-content",
 		},
-		{
-			label: "Action 4",
+		selectionMode: "single",
+		items: [{
+			label: "Crop rotate",
+			iconName: "CropRotate",
+			isSelected: true,
+		}, {
+			label: "Slice",
+			iconName: "VectorDraw",
+		}, {
+			label: "Clone stamp",
+			iconName: "StampClone",
+		}],
+	},
+};
+LongPress.tags = ["!dev"];
+LongPress.parameters = {
+	chromatic: {
+		disableSnapshot: true,
+	},
+};
+
+/**
+ * Action menus can be positioned in four locals relative to the trigger but <em>only one menu can be triggered at a single time</em>.
```

Comment:
I definitely read this as four "low-kuls" first, and figured maybe it was supposed to be "low-cals." Is that a typo for "locales?" Does that word have multiple spellings that I need to remember? 😆

Reply:
Haha yes, it's a typo for locales! 😊